JustLog.podspec

```diff
@@ -13,13 +13,13 @@ Pod::Spec.new do |s|
   s.source           = { :git => 'https://github.com/justeat/JustLog.git', :tag => s.version.to_s }
   s.social_media_url = 'https://twitter.com/justeat_tech'
 
-  s.ios.deployment_target = '10.0'
-  s.tvos.deployment_target = '10.0'
+  s.ios.deployment_target = '13.4'
+  s.tvos.deployment_target = '13.4'
   s.swift_version = '5.0'
   s.pod_target_xcconfig = { 'DEFINES_MODULE' => 'YES' }
 
   s.source_files = 'JustLog/Classes/**/*', 'JustLog/Extensions/**/*'
 
-  s.dependency 'SwiftyBeaver', '~> 1.9.3'
+  s.dependency 'SwiftyBeaver', '1.9.6'
```

Comment:
@albertodebortoli I made it strict because they introduced breaking changes in 1.9.6 (increased min iOS version). To make it safer decided to make it strict. I can make it strict in `Package.swift` too. Oh, you think it's better to add this condition? `"1.9.6"..<"2.0.0"`

Reply:
According to the doc above, if you want it to be strict, in `Package.swift` should have `.exact("1.9.6")`.

Reply:
- [x] done